api/proto/auth/constants.go

```diff
@@ -0,0 +1,34 @@
+package auth
+
+import (
+	"time"
+)
+
+// TODO: copied over from /lib/defaults to limit imports
```

Comment:
Maybe move this defaults.go under the `api` package for now, and then let's get some feedback from other people as well. @r0mant @awly?